backend/src/schema/resolvers/groups.js

```diff
@@ -18,19 +18,25 @@ export default {
         if (isMember === true) {
           groupCypher = `
             MATCH (:User {id: $userId})-[membership:MEMBER_OF]->(group:Group${groupIdCypher})
+            WITH group, membership
```

Comment:
This is important, because after an `With` it works like a filter and is not part of the `MATCH`, see https://neo4j.com/docs/cypher-manual/current/clauses/where/#where-in-operator .